bentoml/_internal/bento/docker/templates/miniconda_debian.j2

```diff
@@ -1,24 +1,10 @@
+{% import '_macros.j2' as common %}
 {% extends "base_debian.j2" %}
 {% set __environment_yml__=expands_bento_path("env", "conda", "environment.yml", bento_path=bento__path) %}
 {% block SETUP_BENTO_ENVARS %}
 {{ super() }}
 
-RUN --mount=type=cache,mode=0777,target=/opt/conda/pkgs bash <<EOF
-set -euxo pipefail
+{{ common.conda_install_python(__python_version__) }}
```

Comment:
let's merge these two macros and the `set __environment_yml__` line above into one macro? since they will always be used together as a group